godot/components/cooking/Applicance.gd

```diff
@@ -50,7 +49,7 @@ func cook(recipe: int) -> bool:
 # - There must be free space on the appliance.
 # - The recipe must be compatible with the appliance.
```

Comment:
I might just be missing it, but where is this check happening, and where are the valid recipes for this appliance stored?

Reply:
see RecipeUtils.gd for this data